src/components/StyleGuide/ColorCard.tsx

```diff
@@ -663,6 +687,30 @@ export const cssVars = {
       dataDarkBgDefaultColor: checkContrast("9.23"),
       textColor: "ui.black",
     },
+    {
+      name: "dark.ui.link.tertiary",
+      dataBlackColor: checkContrast("7.30"),
+      dataWhiteColor: checkContrast("2.88"),
+      dataDarkBgPageColor: checkContrast("6.11"),
+      dataDarkBgDefaultColor: checkContrast("5.33"),
+      textColor: "ui.black",
+    },
+    {
+      name: "dark.ui.link.primary-05",
+      dataBlackColor: ["?", "?", "?"],
+      dataWhiteColor: ["?", "?", "?"],
+      dataDarkBgPageColor: ["?", "?", "?"],
+      dataDarkBgDefaultColor: ["?", "?", "?"],
```

Comment:
What do these question marks mean? Should we add a comment?

Reply:
Yeah, we should.  These colors use transparency and we really are not setup to properly convey how colors like this will impact contrast values.  The question marks were me putting my head in the sand.

Reply:
I added code comments and a note to the Colors Style Guide.